Client is not working in scala 2.13

As you can see in Circle CI the integration against the latest scala version is failing:
https://travis-ci.com/slack-scala-client/slack-scala-client/jobs/287982916

This is due the temporal workaround in: https://github.com/slack-scala-client/slack-scala-client/blob/master/src/main/scala/slack/api/SlackApiClient.scala#L31 that seems not to be any longer valid in this version

Do you see any alternative, as it is not possible to initiate a SlackRtmClient client

Stack trace:
`Exception in thread "main" java.lang.ExceptionInInitializerError
	at slack.api.BlockingSlackApiClient.<init>(BlockingSlackApiClient.scala:38)
	at slack.api.BlockingSlackApiClient$.apply(BlockingSlackApiClient.scala:19)
	at slack.rtm.SlackRtmClient.<init>(SlackRtmClient.scala:33)
	at slack.rtm.SlackRtmClient$.apply(SlackRtmClient.scala:24)
	at acsgh.karma.KarmaBot$.delayedEndpoint$acsgh$karma$KarmaBot$1(KarmaBot.scala:15)
	at acsgh.karma.KarmaBot$delayedInit$body.apply(KarmaBot.scala:13)
	at scala.Function0.apply$mcV$sp(Function0.scala:39)
	at scala.Function0.apply$mcV$sp$(Function0.scala:39)
	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)
	at scala.App.$anonfun$main$1(App.scala:73)
	at scala.App.$anonfun$main$1$adapted(App.scala:73)
	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:553)
	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:551)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:921)
	at scala.App.main(App.scala:73)
	at scala.App.main$(App.scala:71)
	at acsgh.karma.KarmaBot$.main(KarmaBot.scala:13)
	at acsgh.karma.KarmaBot.main(KarmaBot.scala)
Caused by: java.lang.IllegalAccessException: Can not set static final akka.parboiled2.CharPredicate field akka.http.impl.model.parser.CharacterClasses$.strict$minusquery$minuschar$minusnp to akka.parboiled2.CharPredicate$MaskBased
	at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:76)
	at java.base/jdk.internal.reflect.UnsafeFieldAccessorImpl.throwFinalFieldIllegalAccessException(UnsafeFieldAccessorImpl.java:80)
	at java.base/jdk.internal.reflect.UnsafeQualifiedStaticObjectFieldAccessorImpl.set(UnsafeQualifiedStaticObjectFieldAccessorImpl.java:77)
	at java.base/java.lang.reflect.Field.set(Field.java:780)
	at slack.api.SlackApiClient$.<clinit>(SlackApiClient.scala:38)
	... 18 more
`

--- a/src/main/scala/slack/api/SlackApiClient.scala
+++ b/src/main/scala/slack/api/SlackApiClient.scala
@@ -12,7 +12,6 @@ import akka.stream.scaladsl.Source
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.model._
 import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
-import akka.parboiled2.CharPredicate
 import play.api.libs.json._
 
 object SlackApiClient {
@@ -27,16 +26,6 @@ object SlackApiClient {
   private[api] implicit val reactionsResponseFmt = Json.format[ReactionsResponse]
 
   val defaultSlackApiBaseUri = Uri("https://slack.com/api/")
-
-  /* TEMPORARY WORKAROUND - UrlEncode '?' in query string parameters */
-  val charClassesClass = Class.forName("akka.http.impl.model.parser.CharacterClasses$")
-  val charClassesObject = charClassesClass.getField("MODULE$").get(charClassesClass)
-  //  strict-query-char-np
-  val charPredicateField = charClassesObject.getClass.getDeclaredField("strict$minusquery$minuschar$minusnp")
-  charPredicateField.setAccessible(true)
-  val updatedCharPredicate = charPredicateField.get(charClassesObject).asInstanceOf[CharPredicate] -- '?'
-  charPredicateField.set(charClassesObject, updatedCharPredicate)
-  /* END TEMPORARY WORKAROUND */
 
   def apply(token: String, slackApiBaseUri: Uri = defaultSlackApiBaseUri): SlackApiClient = {
     new SlackApiClient(token, slackApiBaseUri)

--- a/src/test/scala/slack/SlackApiClientTest.scala
+++ b/src/test/scala/slack/SlackApiClientTest.scala
@@ -32,7 +32,7 @@ class SlackApiClientTest extends FunSuite with Credentials {
   }
 
   test("send ephemeral with action") {
-    val future = apiClient.postChatEphemeral(channel, "This is an ephemeral", user)
+    val future = apiClient.postChatEphemeral(channel, "This is an ephemeral. How are you?", user)
     val result = Await.result(future, 5.seconds)
 
     println(result)

--- a/src/test/scala/slack/SlackApiClientTest.scala
+++ b/src/test/scala/slack/SlackApiClientTest.scala
@@ -34,7 +34,7 @@ class SlackApiClientTest extends FunSuite with Credentials {
       }
 
       test("send ephemeral with action") {
-        val future = apiClient.postChatEphemeral(channel, "This is an ephemeral. How are you?", user)
+        val future = apiClient.postChatEphemeral(channel, "This is an ephemeral. How are you?", slackUser)
         val result = Await.result(future, 5.seconds)
 
         println(result)

